src/main/java/com/entrevistador/orquestador/infrastructure/adapter/repository/EntrevistaRepository.java

```diff
@@ -10,6 +10,7 @@
 @Repository
 public interface EntrevistaRepository extends ReactiveMongoRepository<EntrevistaEntity, String> {
 
-    @Query(value="{ '_id' : ?0 }", fields="{ 'idHojaDeVidaRag' : 1, 'idInformacionEmpresaRag' : 1}")
-    Mono<RagsIdsDto> findIdHojaDeVidaAndInformacionEmpresaIdById(String id);
+    @Query(value="{ '_id' : ?0 }", fields="{ 'idHojaDeVidaRag' : 1, 'idInformacionEmpresaRag' : 1, 'isHojaDeVidaValida' : 1}")
+    Mono<RagsIdsDto> findIdHojaDeVidaAndInformacionEmpresaAndEsHojaDeVidaValidaIdById(String id);
```

Comment:
Agregar el By luego del find
```suggestion
    Mono<RagsIdsDto> findByIdHojaDeVidaAndInformacionEmpresaAndEsHojaDeVidaValidaIdById(String id);
```

Reply:
el find by esta al final se cambia a español para ser ,ams descriptivo ya que es una projections con @quuiery que no necesita nomenclatura de jpa